dev/com.ibm.ws.jsonb_fat/test-applications/jsonbCDIapp/src/jsonb/cdi/web/JsonbProducer.java

```diff
@@ -0,0 +1,21 @@
+/**
+ *
+ */
+package jsonb.cdi.web;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+
+@ApplicationScoped
+public class JsonbProducer {
+
+    @Produces
+    @ApplicationScoped
+    public Jsonb produceJsonb() {
+        System.out.println("@AGG producer called!!!");
```

Comment:
Did you mean to leave this System.out with your initials in the test code?  Otherwise, the changes look good.